README.md

```diff
@@ -1,11 +1,18 @@
 # Pristine
 
-Pristine is an open source repository in its original condition. It is meant to be a starting point and a place to find resources to augment your own open source repositories.
+Pristine is an open source repository in its original condition. It's goal is to be a starting point for open source repositories wanting to follow Documentation Driven Development. It is also a place for existing projects to find resources to augment current documentation.
 
+## Documentation Driven Development
+
+There are many ways to drive development but with open source specifically, starting at the README gives a middle ground between technical specifications and non-technical. This promotes open source contribution and engagement.
+
+> By the same principle a beautifully crafted library with no documentation is also damn near worthless. If your software solves the wrong problem or nobody can figure out how to use it, there’s something very bad going on.
+
+- [Readme Driven Development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html) by Tom Preson-Werner
 
 ## Getting Started
 
-To get started, you will need a [fork of the repository](https://help.github.com/articles/fork-a-repo/). Then you can edit this file and delete everything above this line.
+To get started, [fork](https://help.github.com/articles/fork-a-repo/) or [duplicate](https://help.github.com/articles/duplicating-a-repository/) the repository. Then edit this file and delete everything above this line.
```

Comment:
Should the following paragraph `Conventions and Specifications` also be removed?

Reply:
Agreed